apps/explorer/tsconfig.json

```diff
@@ -19,7 +19,10 @@
     "target": "ES2022",
     "module": "ES2022",
     "useDefineForClassFields": false,
-    "lib": ["ES2022", "dom"]
+    "lib": ["ES2022", "dom"],
+    "paths": {
+      "tslib": ["./node_modules/tslib/tslib.d.ts"]
+    }
```

Comment:
Are we sure that tslib definition is needed?

Reply:
Yes, or else we gonna have problems with the linter complaining about it.